public/reorderSiteAwards.php

```diff
@@ -61,30 +186,39 @@ function RenderAwardOrderTable(string $title, array $awards, int &$counter): voi
 
                 sanitize_outputs(
                     $awardTitle,
-                    $awardGameConsole
+                    $awardGameConsole,
+                    $awardType,
+                    $awardData,
+                    $awardDataExtra,
                 );
 
                 if ($awardType == AwardType::AchievementUnlocksYield) {
                     $awardTitle = "Achievements Earned by Others";
                 } elseif ($awardType == AwardType::AchievementPointsYield) {
                     $awardTitle = "Achievement Points Earned by Others";
-                // } elseif ($awardType == AwardType::Referrals) {
-                //     $awardTitle = "Referral Award";
                 } elseif ($awardType == AwardType::PatreonSupporter) {
                     $awardTitle = "Patreon Supporter";
                 }
 
+                echo "<tr draggable='true' class='cursor-grab transition' ondragstart='handleRowDragStart(event)' ondragenter='handleRowDragEnter(event)' ondragleave='handleRowDragLeave(event)' ondragover='handleRowDragOver(event)' ondragend='handleRowDragEnd(event)' ondrop='handleRowDrop(event)'>";
                 echo "<td>";
                 RenderAward($award, 48, false);
                 echo "</td>";
                 echo "<td>$awardTitle</td>";
                 echo "<td style=\"white-space: nowrap\"><span class='smalldate'>$awardDate</span><br></td>";
-                echo "<td><input class='displayorderedit' id='$counter' type='text' value='$awardDisplayOrder' onchange=\"updateAwardDisplayOrder('$awardType', '$awardData', '$awardDataExtra', '$counter')\" size='3' /></td>";
+                echo "<td><input class='displayorderedit' data-award-type='$humanReadableAwardType' id='$counter' type='text' value='$awardDisplayOrder' size='3' /></td>";
+                echo "<input type='hidden' name='type' value='$awardType'>";
+                echo "<input type='hidden' name='data' value='$awardData'>";
+                echo "<input type='hidden' name='extra' value='$awardDataExtra'>";
 
                 echo "</tr>\n";
                 $counter++;
             }
             echo "</tbody></table>\n";
+
+            echo "<div class='flex w-full justify-end'>";
+            echo "<button onclick='updateAllAwardsDisplayOrder()'>Save</button>";
```

Comment:
Having a Save button in each section is misleading. I can make changes to my Game Awards, but not Save, then go hide a Site Award and when I Save that, the Game Awards get saved too.